requirements.in

```diff
@@ -0,0 +1,7 @@
+autopep8
```

Comment:
could you please call this `requirements-test.in`? would make it more descriptive, what this packages are for